content/blog/2018-04-22-essurvey.md

```diff
@@ -0,0 +1,337 @@
+date: 2018-01-05
+categories: blog
+topicid: 925
+output: md_document
+tags:
+- r
+- community
+- data
+- review
+- onboarding
+- package
+- data-extraction
+- social-science
+---
+
+
+<!-- # see for blogpost https://github.com/ropensci/roweb2#contributing-a-blog-post -->
+Introduction
+------------
+
+I never thought that I'd be programming software in my career. I started
+using R little over 2 years now and it's been one of the most important
+decisions in my career. Secluded in a small academic office with no one
+to discuss/interact about my new hobbie, I started searching the web for
+tutorials and packages. After getting to know how amazing and nurturing
+the R community is, it made me want to become a data scientist. So I set
+out to do it. Through out the journey I repeatedly found myself using
+the [European Social Survey](http://www.europeansocialsurvey.org/), a
+really neat dataset that collects informations on attitudes, beliefs and
+behaviour patterns of diverse populations in more than thirty European
+nations since 2002.
+
+After seeing a niche in the R package community, I created the package
+`essurvey` (previously `ess` in CRAN) to access this data easily from R.
+The package was accepted in CRAN in September 2017 and was well received
+among social scientists.
+
+Over a month ago I submitted the package to rOpensci, intimidated but
+very excited about the peer review process. To my surprise, the process
+was enriching, respectful and transparent, unlike my previous experience
+in academic research.
+
+First steps
+-----------
+
+Using the `essurvey` package is fairly easy. There are are two main
+families of functions: `import_*` and `show_*`. They each complement
+each other and allow the user to almost never have to go to the European
+Social Survey (ESS) website. The only scenario where you need to enter
+the ESS website is to validate your email. If you haven't registered,
+create an account at <http://www.europeansocialsurvey.org/user/new> and
+validate through your email.
+
+Given that some `essurvey` functions require your email address, this
+vignette will use a fake email but everything should work accordingly if
+you registered with the ESS.
+
+You can install the development version with
+
+    # install.packages("devtools")
+    devtools::install_github("ropensci/essurvey")
+
+or the stable version from CRAN with
+
+    install.packages("essurvey")
+
+Let's load the CRAN version and load it together with the `tidyverse`
+set of packages to do some data manipulation.
+
+    library(essurvey)
+    library(tidyverse)
+
+Downloading the ESS data requires validating your email everytime you
+download data. We can set our email as an environment variable with
+`set_email`.
+
+    set_email("your@email.com")
+
+Once that's executed you can delete the previous line and any `import_*`
+call will look for the email automatically, stored as an environment
+variable.
+
+Exploring country data

```

Comment:
Possibly before the "Exploring country data" section, the post would benefit immensely from an explanation / definitions of the nature, scope and terminology of ESS data. This would help people imagine how they might use your pkg to do cool things. 

More importantly, I found the use of these terms confusing and my reading slowed down as I had to make some inferences - rounds, waves, questions, scales, categories. Definition or explanation of these terms in one paragraph about nature of ESS data allows reader to picture what you are doing. Until I understood clearly what x-axis labels ESS1, ESS2 etc were, I had to go back in the text and code snippets to figure them out.